packages/core/utils/src/relativeBundlePath.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import type {Bundle} from '@parcel/types';
+import path from 'path';
+import nullthrows from 'nullthrows';
+
+export function relativeBundlePath(from: Bundle, to: Bundle) {
```

Comment:
This is really an URL rather than a path (because it returns a URL with forward slashes also on Windows).